gui/components/auth/forms/registrationForm.tsx

```diff
@@ -1,24 +1,26 @@
-import {Image, StyleSheet, Text, TextInput, TouchableOpacity, View} from "react-native";
+import { Image, StyleSheet, Text, TextInput, TouchableOpacity, View } from "react-native";
 import globalStyles from "../../../styles/global";
 import * as React from "react";
-import {useAuth} from "../../../context/auth";
+import { useAuth } from "../../../context/auth";
 import * as ImagePicker from "expo-image-picker";
-import {postRequest} from "../../../util/ajax";
-import {z} from "zod";
-import {useState} from "react";
+import { postRequest } from "../../../util/ajax";
+import { z } from "zod";
+import { useState } from "react";
+import { useRouter } from "expo-router";
 
 
 
 const registrationSchema = z.object({
     name: z.string().min(2, { message: "Name must be at least 2 characters" }),
     email: z.string().email({ message: "Invalid email address" }),
-    password: z.string().min(16, { message: "Password must be at least 16 characters" }),
+    password: z.string().min(8, { message: "Password must be at least 16 characters" }),
```

Comment:
Password minimum is 16 right now.

Reply:
If changing to the text will need to be changed in the message as well and I'll need to change it in all of the BDD registration tests.

Reply:
Yes, there should be an edit to the [tests](../../tests) directory as well.